### PR TITLE
[#61]  Create A  PageNavheader Component

### PR DESCRIPTION
**Pull request by goodylove**
Trello:  https://trello.com/c/5Ry4o8C2/61-create-a-pagenavheader-component

### CODE REVIEW

File: src/components/PageNavHeader/pageNavHeader.test.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  test("should Go back to previous page on click on the left-caret icon", async () => {
+    const user = userEvent.setup();
+
+    renderComponent();
+
+    const iconElement = screen.getByTestId("icon");
+    await user.click(iconElement);
+
+    expect(navigate).toHaveBeenCalledWith(-1);
+    console.log(navigate);
```
Comment (stanleyogada) [inline src/components/PageNavHeader/pageNavHeader.test.tsx:37]: Forgotten log

File: src/components/PageNavHeader/pageNavHeader.test.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+describe("render pageNavHeader correctly", () => {
+  test("should display the header correctly", () => {
+    renderComponent();
+    const headingElement = screen.getByRole("heading");
+    const arrowElement = screen.getByTitle("left-caret");
+
+    expect(arrowElement).toBeInTheDocument();
+    expect(headingElement).toBeInTheDocument();
+  });
```
Comment (stanleyogada) [inline src/components/PageNavHeader/pageNavHeader.test.tsx:19]: Remove render test

File: src/components/PageNavHeader/pageNavHeader.test.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+  test("should Have the padding styles provided correctly", () => {
+    renderComponent();
+    const divElement = screen.getByTitle("left-caret");
+
+    expect(divElement.parentNode).toHaveStyle("padding: 50px 0px 15px 29px");
+  });
```
Comment (stanleyogada) [inline src/components/PageNavHeader/pageNavHeader.test.tsx:26]: Remove this test

**Review by stanleyogada — CHANGES_REQUESTED** [review]

File: src/components/PageNavHeader/pageNavHeader.test.tsx (hunk 2 above)
Comment (goodylove) [reply src/components/PageNavHeader/pageNavHeader.test.tsx:19]: but this is among the test case  you add on trello

**Review by stanleyogada — APPROVED** [review]

File: src/components/PageNavHeader/pageNavHeader.test.tsx (hunk 1 above)
Comment (stanleyogada) [reply src/components/PageNavHeader/pageNavHeader.test.tsx:37]: Fix this

**Review by stanleyogada — CHANGES_REQUESTED** [review]

File: src/components/PageNavHeader/pageNavHeader.test.tsx (hunk 1 above)
Comment (goodylove) [reply src/components/PageNavHeader/pageNavHeader.test.tsx:37]: sure . but approve am on it

**Comment by goodylove** [comment]
i thought you just approved this now ?

File: src/components/PageNavHeader/pageNavHeader.test.tsx (hunk 1 above)
Comment (stanleyogada) [reply src/components/PageNavHeader/pageNavHeader.test.tsx:37]: Why are you replying here? Push your commit to fix instead

**Pull request merged**